server/src/main/java/org/candlepin/dto/StandardTranslator.java

```diff
@@ -142,7 +140,7 @@ public StandardTranslator(ConsumerTypeCurator consumerTypeCurator,
         this.registerTranslator(
             new BrandingTranslator(), Branding.class, BrandingDTO.class);
         this.registerTranslator(
-            new ProductBrandingTranslator(), ProductBranding.class, BrandingDTO.class);
+            new BrandingTranslator(), Branding.class, BrandingDTO.class);
```

Comment:
This whole line can be removed, since the one above it does exactly the same thing. It was only temporarily added for the short period of the workaround of having 2 brandings (Branding and ProductBranding).

Reply:
good catch, Correcting it.